core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/conditions/TokenBucketRetryCondition.java

```diff
@@ -122,9 +125,17 @@ public boolean shouldRetry(RetryPolicyContext context) {
             context.executionAttributes().putAttribute(LAST_ACQUIRED_CAPACITY, c);
             context.executionAttributes().putAttribute(RETRY_COUNT_OF_LAST_CAPACITY_ACQUISITION,
                                                        context.retriesAttempted());
+            log.trace(() -> "Successfully acquired token bucket capacity to retry this request. "
```

Comment:
`debug`?

Reply:
This seemed like a bit much for debug, since we're already logging when there wasn't enough capacity. I was thinking if we really needed to debug this, we can ask them to enable trace for this one class.